Add instructions to use Cargo to set GODOT4_BIN to relative path. (#20)
godot-rust/gdext-book: src/toolchain/godot-version.md

Consult the [setup page][setup-llvm] for more information.

> ### Setting `GODOT4_BIN` to a Relative Path

Reviewer: ```suggestion
### Setting `GODOT4_BIN` to a relative path
```

> If you have multiple Godot workspaces on a machine, you may want a workspace-independent method of setting the `GODOT4_BIN` environment variable
> so the matching Godot editor binary for that workspace is always used in the build process, without having to set `GODOT4_BIN` differently for each
> location.

Reviewer: I'd probably split into multiple sentences:
```suggestion
If you have multiple Godot workspaces on a machine, you may want a workspace-independent method of setting the `GODOT4_BIN` environment variable.
This way, the matching Godot editor binary for that workspace is always used in the build process, without having to set `GODOT4_BIN` differently for each
location.
```

You can do this by configuring Cargo to set `GODOT4_BIN` to a relative path for you, in `.cargo/config.toml`.

In the root of your Rust project, create `.cargo/config.toml` with the example content shown below, modifying the editor path as needed to find

> your binary. The path you set will be resolved relative to the location of the `.cargo` directory.

Reviewer: ```suggestion
your binary. The path you set will be resolved relatively to the location of the `.cargo` directory.
```
